website/components/features-list/style.module.css

```diff
@@ -0,0 +1,112 @@
+.featureListContainer {
+  background-color: #000;
+  padding-top: 128px;
+  background-position: right top, left bottom;
+  background-repeat: no-repeat;
+  color: #fff;
+}
+
+.contentWrapper {
+  composes: g-grid-container from global;
+}
+
+.featureContainer {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+
+  & .featureTextContainer {
+    display: flex;
+    margin-right: 60px;
```

Comment:
This is only needed when they are side-by-side -- mind wrapping this in the same breakpoint that you use to flip the flex-direction?

Gives a little more room for the content.

<img width="1237" alt="Screen Shot 2021-09-21 at 12 24 50 PM" src="https://user-images.githubusercontent.com/2105067/134234743-dc5334b4-58fe-4187-bf0b-ac7eb6c9a0d4.png">